scripts/build-and-test.sh

```diff
@@ -38,11 +44,11 @@ if [[ $VARIANT == "coverage" ]]; then
   lcov --no-external -c -i -d . -o baseline.info > /dev/null
 
   # Perform test
-  run_tests_with_gdb
+  run_tests
 
   # Run autobahn tests
-  export SERVER=`find . -name "websocket-echo"`
-  nohup scripts/run-with-gdb.sh $SERVER&
```

Comment:
I've noticed some stalls some time, which were the reason for the (hamfisted) extra kill invocation (one possible cause of the issue).